CHANGELOG.md

```diff
@@ -0,0 +1,177 @@
+# Changelog
+
+## [Unreleased]
+### Added
+- Add Dockerfile for `radicle-cli`
+- Re-organize binaries into own crate and move all sub-commands to `rad` binary 
```

Comment:
Who is the Changelog directed at? I ask because this appears dev facing

Reply:
Ya, agree. So I tried to find a middle ground between a changelog for devs / technical writers and end users. But I guess your comment and the reactions to it smell like necessary simplification.